adapters/goal.ts

```diff
@@ -12,7 +12,7 @@ const goalSiteConfig = {
 
 export const fetchGoals = async (): Promise<Goal[]> => {
   const goals: Goal[] = await axios
-    .get(`${API.GOAL_BASE_URL}/goal/`)
+    .get(`${API.GOAL_BASE_URL}/api/v1/goal/`)
```

Comment:
let's include `/api/v1/` in GOAL_BASE_URL

Reply:
I think `/api` in the GOAL_BASE_URL will do as v1 can change to v2 someday